data/options.nix

```diff
@@ -176,6 +176,14 @@ in
       '';
     };
 
+    targetPort = mkOption {
+      type = int;
```

Comment:
I prefer `nullOr int` here, and instead of the magic checking for port 22, always specify -p/-P if targetPort is set regardless of the port number. Can you make that adjustment, please? :)